changelog/5266.misc.rst

```diff
@@ -7,3 +7,6 @@ include in your pipeline.
 Use ``DIETClassifier`` instead of ``SklearnIntentClassifier``.
 
 Use ``TEDPolicy`` instead of ``KerasPolicy``.
+
+Properties ``Component.provides`` and ``Component.requires`` are deprecated.
```

Comment:
This shouldn't be misc (as this is relevant for the user) - I think the note should be `5266.removal.rst`